package.json

```diff
@@ -5,7 +5,7 @@
   "description": "Fastish HTML",
   "main": "index.js",
   "engines": {
-    "node": "^10.0.0",
+    "node": "^12.0.0",
```

Comment:
To increase resiliency, it might be worth allowing the current and the next LTS version (`^12 || ^14`) in a future PR.

Reply:
Some developers reported on slack they are no longer able to use amphtml-validator with nodejs 10.0. And they cannot upgrade to 12.0 soon.

Can we update this to support >=10.0 ?

Sorry, I do not have much background about this upgrade.

Reply:
+1. FWIW, v10 is not officially EOL until 2021-04-30 per https://nodejs.org/en/about/releases/, so if it doesn't break anything major in amphtml-validator, it would be nice to continue supporting it.

Reply:
It sounds reasonable to support all currently Active LTS versions (as of today, 10 and 12). So maybe this should be `^10 || ^12 || ^14`?

(This is different from `>= 10` because we want to skip major versions that will never be an LTS version.)

Reply:
PR here for `>=10.0.0`, could update to the suggestion above. https://github.com/ampproject/amphtml/pull/25208

Reply:
To build future resiliency, we can have renovate auto-update the `engines` section of all our `package.json` files: https://docs.renovatebot.com/node/

I'll file a tracking issue.